ui/app/src/components/Icons/LevelDescriptor.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2007-2020 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import { createSvgIcon } from '@material-ui/core/utils';
+
+export default createSvgIcon(
+  <>
+    <g clipPath="url(#clip0)">
+      <path d="M8.74342 4.4647C8.74342 3.9947 9.31342 3.7547 9.64842 4.0897L11.8884 6.3297C12.0834 6.5247 12.0834 6.8397 11.8884 7.0347L9.64842 9.2747C9.31342 9.6147 8.74342 9.3747 8.74342 8.9047C8.74342 8.7647 8.79842 8.6297 8.89842 8.5297L10.7434 6.6847L8.89842 4.8397C8.79842 4.7397 8.74342 4.6047 8.74342 4.4647ZM5.74342 5.1847V4.3897C5.74342 3.9447 6.28342 3.7197 6.59842 4.0347L8.89342 6.3297C9.08842 6.5247 9.08842 6.8397 8.89342 7.0347L6.59842 9.3297C6.28342 9.6447 5.74342 9.4197 5.74342 8.9747V8.1347C3.24342 8.1347 1.49342 8.9347 0.243418 10.6847C0.743418 8.1847 2.24342 5.6847 5.74342 5.1847Z" />
+    </g>
+    <defs>
+      <clipPath id="clip0">
+        <rect width="12" height="12" fill="white" transform="translate(0.0344238 0.68457)" />
+      </clipPath>
+    </defs>
```

Comment:
Can we get rid of everything but the paths?

Reply:
yes